datadog-oci-orm/metrics-setup/schema.yaml

```diff
@@ -37,22 +41,37 @@ variables:
     default: datadog-metrics
   create_vcn:
     title: Create VCN
-    description: Optional variable to create virtual network for the setup. Otherwise, choose an existing subnet from VCN
+    description: Create virtual network (VCN) for the function setup. If using existing networking resources, de-select "Create VCN" and select the relevant VCN and subnet to use.
     type: boolean
     default: true
 
+  compartment_ocid:
+    # prepopulates available values for compartment
+    required: true
+    type: oci:identity:compartment:id
+    description: The compartment OCID to where this terraform stack is created and the following networking resources, function, and connecter hub will be deployed.
+
 # VCN
   vcnCompartment:
     # prepopulates available values for compartment
     type: oci:identity:compartment:id
+    visible:
+      not:
+        - ${create_vcn}
   existingVcn:
     type: oci:core:vcn:id
+    visible:
+      not:
+        - ${create_vcn}    
```

Comment:
Nice. Since you have already added the visibility changes, I think it would be beneficial to hide the tenancy variables as well in this PR itself.